openapi-v0.yaml

```diff
@@ -47,80 +47,105 @@ components:
         identityName:
           type: string
           example: morning-briefing
+          description: Human-readable, unique identifier of the newsletter in kebab-case
         name:
           type: string
           example: First Edition
+          description: Name of the newsletter
         cancelled:
           type: boolean
           example: false
+          description: Indicates if the newsletter has been cancelled. This prevents it from being actively promoted across the site and prevents sign-ups and unsubscribes.
         restricted:
           type: boolean
           example: false
+          description: Indicates if the newsletter is restricted (e.g. not available to the public, requiring payment/subscription, or requiring a registered account).
         paused:
           type: boolean
           example: false
+          description: Indicates if the newsletter is paused (temporarily not running). This prevents it from being actively promoted across the site and prevents most sign-ups.
         emailConfirmation:
           type: boolean
           example: false
+          description: Indicates if the newsletter sign up process requires an email-confirmation step in order to subscribe.
         brazeNewsletterName:
           type: string
           example: Editorial_MorningBriefingUK
+          description: The newsletter name identifier used to create segments in Braze
         brazeSubscribeAttributeName:
           type: string
           example: MorningBriefingUk_Subscribe_Email
+          description: Used for custom attributes in Braze. Detemines whether a user is a subscriber to a newsletter or not.
         brazeSubscribeEventNamePrefix:
           type: string
           example: morning_briefing_uk
+          description: Used for custom events in Braze. Detemines whether a user is a subscriber to a newsletter or not.
         theme:
           type: string
           example: News
+          description: Describes the newsletter theme (similar to a pillar) which is a top level category.
         group:
           type: string
           example: News in brief
+          description: Describes the newsletter group. Used in the all newsletters page as a secondary level grouping to the theme.
         description:
           type: string
           example: Archie Bland and Nimo Omer take you through the top stories and what they mean, free every weekday morning
+          description: Provides a short description of the newsletter for promotion across the site.
         regionFocus:
           type: string
           example: UK
+          description: Indicates whether the newsletter content is tailored towards a particular region.
```

Comment:
I was thinking of "Edition" as in the UK Edition or AUS Edition of theguardian.com - I.E. we might say "regionFocused" should ideally be  "uk", "us" or "aus" so they can be associated with an Edition of the site. However, thinking again, it might be best not to mention that - as things stand, there is no explicit link between the concepts.

Reply:
Where a value can have one of a pre-determined set of values (rather than being free-form text), it's probably worth mentioning what the options (currently) are, as it forms part of the definition.  Interesting question about whether a newsletter that isn't specifically focussed to UK, AU(S) or US should be left blank or assigned to INTL (as per David's suggestion).  I think I quite like INTL rather than blank, as it's clear then that it's not just an accidental omission.

Reply:
Good point!  I was forgetting that we're currently documenting what we have, rather than what we want!